Auth/README.md

```diff
@@ -49,6 +50,8 @@ Auth guides at the following links:
 - [Google](https://firebase.google.com/docs/auth/ios/google-signin#before_you_begin)
 - [Facebook](https://firebase.google.com/docs/auth/ios/facebook-login#before_you_begin)
 - [Phone](https://firebase.google.com/docs/auth/ios/phone-auth#before_you_begin)
+- [Sign in with Apple](https://firebase.google.com/docs/auth/ios/apple#before_you_begin)
+  - For Sign in with Apple, read the *Comply with Apple anonymized data requirements* section as well. 
```

Comment:
Insert link to https://firebase.google.com/docs/auth/ios/apple#comply-with-apple-anonymized-data-requirements